### PR TITLE
Put HomeNavMenuDropdownButton in its own file [Resolves #74]

### LINKED ISSUE #74: Put HomeNavMenuDropdownButton in its own file
and put associated styles in their own file.

### PR DESCRIPTION
**Pull request by jason00111**
- Refactored into ES6 class

### CODE REVIEW

File: src/components/HomeNav.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import React, { Component } from 'react'
-import HomeNavDropdownMenu from './HomeNavDropdownMenu'
 import HomeNavHamburger from './HomeNavHamburger'
```
Comment (jamestewartjr) [inline src/components/HomeNav.js]: New line to separate library imports from component imports.

File: src/components/HomeNav.js [hunk 2]
```diff
@@ -40,38 +39,4 @@ class HomeNav extends Component {
...
-    return <div>
-      <a onClick={this.toggleMenu} href='#'>More
-          <div className='more-triangle' />
-      </a>
-      <HomeNavDropdownMenu isOpen={this.state.isOpen} />
-    </div>
-  }
-})
-
 export default HomeNav
```
Comment (jamestewartjr) [inline src/components/HomeNav.js]: JRob has requested that "export default" is included inline with the declaration of the component.

**Comment by jamestewartjr** [comment]
Is the "more triangle" supposed to trigger a new menu?

**Comment by Anasauce** [comment]
@jamestewartjr we didn't write the CSS for "more-triangle" we just refactored the already written code for that component. We saw that it was the only CSS class that was called in that file so it was included in it's corresponding CSS file. The button however, didn't work prior to refactoring and doesn't work now. So we are going to make a separate issue to fix that bug.

File: src/components/landing/navigation/DropdownButton.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  constructor( props ) {
+    super( props )
+
+    this.state = {
+      isOpen: false
+    }
+  }
+
+  toggleMenu( event ) {
+    this.setState( { isOpen: !this.state.isOpen } )
```
Comment (jrob8577) [inline src/components/landing/navigation/DropdownButton.js]: Please remove spaces between `(` and `{`

File: src/components/landing/navigation/DropdownButton.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  onClose() {
+    this.setState( { isOpen: false } )
```
Comment (jrob8577) [inline src/components/landing/navigation/DropdownButton.js]: Please remove spaces between `(` and `{`

**Pull request merged**